ci/ci/ci.py

```diff
@@ -214,13 +214,29 @@ def refresh_deploy_jobs(jobs):
                 try_to_cancel_job(job)
     prs.refresh_from_deploy_jobs(latest_jobs)
 
+
+@app.route('/force_retest_flat', methods=['POST'])
+def force_retest_flat():
+    d = request.json
```

Comment:
I am honestly surprised this works. I thought you'd have to use [`request.form`](http://flask.pocoo.org/docs/1.0/quickstart/#the-request-object)

Reply:
how do I test this?

Reply:
I promise to make it less of a PITA ASAP.

Reply:
![image](https://user-images.githubusercontent.com/10562794/51632775-126b1b80-1f1e-11e9-8896-7b5ca47b7a92.png)